DataSetExplorer/DataSetBuilder/Model/DataSetProject.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DataSetExplorer.DataSetBuilder.Model
+{
+    public class DataSetProject
+    {
+        internal readonly string _name;
```

Comment:
We use the "_" prefix to signal private fields, and these are internal.

Since we will need to map them to DTOs, we should stick with properties with exposed getters and private setters.